copy_edit.py

```diff
@@ -92,7 +92,7 @@ def run(self, edit):
 			sel_strings = iter(selection_strings)
 			string = next(sel_strings)
 			for sel in self.view.sel():
-				replace_region = sublime.Region(self.view.line(sel.begin()).begin()) if string[1] else sel
+				replace_region = sublime.Region(self.view.line(sel.begin()).begin()) if string[1] and sel.size() == 0 else sel
```

Comment:
What does this fix?

Reply:
Obviously, it fixes test TN 8, as commit message says. :-) 
```
TN 8
DA‘1>‘’<
2’
CO copy
DA‘>‘1
’<2’
CO paste
CR‘1
>‘’<2’
```
If you still don't get it [i.e. how to read this tests' notation], here is the explanation:
```
DA‘1>‘’<
2’
```
means initialize DAta (or DAno in Russian) with:
```
1|
2
```
(here `|` means cursor/caret)

`CO copy` means run COmmand copy (like `view.run_command("copy")`).
```
DA‘>‘1
’<2’
```
means replace DAta with:
```
|>1
<|2
```
(here `|>` means selection start, and '<|' means selection end)

`CO paste` means run COmmand paste (like `view.run_command("paste")`).

And at last
```
CR‘1
>‘’<2’
```
means that Correct Result is should be the following:
```
1
|2
```

Before this commit the result was this:
```
1
1
|2
```